.github/workflows/code_analysis_and_tests.yml

```
@@ -25,6 +25,9 @@ jobs:
       - name: Get Flutter dependencies
         run: flutter pub get
 
+      - name: Generate Flutter files
+        run: flutter packages pub run build_runner build
```

We should run this command with `--delete-conflicting-outputs` param.

```suggestion
        run: flutter packages pub run build_runner build --delete-conflicting-outputs`
```